### PR TITLE
feat(dropdown/basic): allow custom classname on button

### PR DESCRIPTION
**Pull request by ruben-martin-lozano**
Allow custom button className

### CODE REVIEW

File: components/dropdown/basic/src/index.js [hunk 1]
```diff
@@ -134,13 +134,17 @@ class DropdownBasic extends Component {
     const wrapperClassName = cx('sui-DropdownBasic', {
       'is-expanded': expanded
     })
+    const buttonWrapperClassName = cx('sui-DropdownBasic-buttonWrap', {
+      [button.className]: !!button.className
```
Comment (carlosvillu) [inline components/dropdown/basic/src/index.js]: Classname brake the encapsulation of this component.

You must use an enum prop and apply the inner class properly depend on the enum value.

Comment (kikoruiz) [reply components/dropdown/basic/src/index.js]: Just for info... why do you need a custom classname?

Comment (miduga) [reply components/dropdown/basic/src/index.js]: He needs a custom className to just use a `re-Select` class in order to give to the dropdown the look'n'feel of Fotocasa. This is not ideal, as the re-Select itself should be a component instead just classnames to be used.

Comment (kikoruiz) [reply components/dropdown/basic/src/index.js]: As you say, it's not the ideal, and as far as I know, we've been taken a different approach for this specific cases by overriding the classnames from our sites containers... So, I don't know if it's a door we should open, aren't you happy by overriding classnames?

**Comment by ruben-martin-lozano** [comment]
Not necesary at this moment.

**Pull request closed without merging**